docs/advanced-features/compiler.md

```diff
@@ -331,30 +361,77 @@ module.exports = {
 }
 ```
 
-Cause this code:
+The above config will transform your code as follows:
```

Comment:
I made the examples more consistent by following this pattern throughout this page:

> ```js
> // next.config.js
> ...
> ```
>
> The above config will transform your code as follows:
>
> ```js
> // Before
> ...
>
> // After
> ...
> ```